mm2src/lp_stats.rs

```diff
@@ -0,0 +1,334 @@
+/// The module is responsible for mm2 network stats collection
+///
+use common::executor::{spawn, Timer};
+use common::mm_ctx::MmArc;
+use common::mm_error::prelude::*;
+use common::{log, now_ms, HttpStatusCode};
+use derive_more::Display;
+use http::StatusCode;
+use mm2_libp2p::atomicdex_behaviour::parse_relay_address;
+use mm2_libp2p::{encode_message, PeerId};
+use serde_json::{self as json, Value as Json};
+use std::collections::{HashMap, HashSet};
+use std::net::ToSocketAddrs;
+
+use crate::mm2::lp_network::{add_peer_addresses, request_peers, P2PRequest, PeerDecodedResponse};
```

Comment:
If we import functions from `lp_network` here, how about returning `lp_ports` them to the `lp_network`?